### ISSUE TITLE
Rendering on Android starts ok, degrades while playing

### ISSUE THREAD

**Issue by YellowLabrador** [vpinball/vpinball#1028]
**Describe the bug**
Rendering on Android starts ok, degrades while playing

**To Reproduce**
Launch any table and start playing. I tested Attack on Mars, exampleTable and strippedTable

**Expected behavior**
Graphics get "dirty" as you play and objects move on the screen. In the screenshots I took you can see how the area around the flippers are when you launch the game and what they look like after moving them a few times. The same thing happens while you play and the ball moves around the table, wherever the ball goes the graphics gets worse and worse.

**Screenshots**
![after launching](https://github.com/vpinball/vpinball/assets/132696731/54cadf3d-d744-4c9c-9f33-270d080bb2fa)
![after moving the flippers a few times](https://github.com/vpinball/vpinball/assets/132696731/78851dfa-612e-4e6f-aade-2a2b6012be78)

**Versions & Settings**
 - VPX: standalone (e054d33)
 - Android 13, One UI 5.1.1 (Samsung)

**Comment by jsm174** [comment]
Hello. I'm having trouble identifying that hash. Do you have the exact name of the download? It should have a build number as well.

Was there ever a version that rendered okay for you?

**Comment by YellowLabrador** [comment]
Here is the `git log` from my build

```
commit 09950cd35ac0b9d3a8afaab41a94c9cb7e054d33 (HEAD -> standalone, origin/standalone)
Author: Jason Millard <jsm174@gmail.com>
Date:   Sat Oct 28 21:49:05 2023 -0400

    misc: standalone builds for macos, ios, tvos, android, linux, and rpi
```

I sync'ed and built it a few times starting a couple of weeks ago. It never worked well, I always experienced this same rendering issue.

**Comment by YellowLabrador** [comment]
Here is what happens when I switch the view layout mode from Camera to Window and back. The first screenshot is from launching VPX. It looks fine at first, the other two are from changing the layout mode.

![Screenshot_20231029_202108_VPinballX_GL](https://github.com/vpinball/vpinball/assets/132696731/1556ce28-05ac-48b0-b59d-3014061f7eeb)
![Screenshot_20231029_202131_VPinballX_GL](https://github.com/vpinball/vpinball/assets/132696731/80d895e7-ac33-4149-a345-fdfa5a8bead1)
![Screenshot_20231029_202202_VPinballX_GL](https://github.com/vpinball/vpinball/assets/132696731/fbc6fcb4-3aeb-4cdf-9c8e-8421f444a7d9)

**Comment by jsm174** [comment]
Thanks. The number you posted was from the end of the SHA instead of the beginning so thats why I couldn't find it.

Unfortunately Android devices are all over that map with GLES support.


I have a Pixel 7 test device.

I just installed the latest from the CI. I'm not seeing those issues:

https://github.com/vpinball/vpinball/assets/1197137/60a2eca2-b765-4c37-9025-e483858985f4

Attack from Mars LE v5_VPX8.vpx:

https://github.com/vpinball/vpinball/assets/1197137/c461946d-4690-4901-b474-267042ccdc06

**Comment by YellowLabrador** [comment]
Fair enough. I tried the same build on a Pixel Tablet and it worked fine as well, both in portrait and landscape. The issue is on Samsung (tablets). Any logs or other info I can provide to help diagnose the problem?

**Comment by jsm174** [comment]
It's not for the faint of heart, but maybe you could try RenderDoc, and see if anything jumps out. It helped us figure out a long standing bug where our LiveUI was flickering only in GLES.

**Comment by jsm174** [comment]
Can you see if this is still an issue on 10.8.1.5115?
